### ISSUE TITLE
AMD Ryzen 9 5900X keeps crashing when running the miner

### ISSUE THREAD

**Issue by zeybor** [WyvernTKC/cpuminer-gr-avx2#137]
AMD Ryzen 9 5900X keeps crashing when running the miner on windows 10. Trying to run it the first in order to tune the miner, however after running for a few minutes, system crashes and shuts down. Is this a known issue, what could be the reason?

**Comment by michal-zurkowski** [comment]
That is very likely just unstable cpu.
Try running 20-30 runs of linpack Xtreme with at least 8gb of datasets to make sure cpu is at least somehow stable.

**Comment by zeybor** [comment]
> That is very likely just unstable cpu. Try running 20-30 runs of linpack Xtreme with at least 8gb of datasets to make sure cpu is at least somehow stable.

Thank you. I will try to run the stress test

**Comment by zeybor** [comment]
>

**Comment by zeybor** [comment]
> 

indeed, system crashes pretty much straight away. heat quickly goes above 120degress. i will look into it...
